### PR TITLE
LPS-102706 Delete the corresponding expando rows as well upon deleting the MBMessages

### PR DESCRIPTION
**Pull request by hudakl**
Hi Roberto,

Could you please review my soltuion suggestion for LPS-102706?

Thanks,
Laci

### CODE REVIEW

**Comment by robertoDiaz** [comment]
ci:test:sf

**Comment by robertoDiaz** [comment]
ci:test:relevant

File: modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBThreadLocalServiceImpl.java [hunk 1]
```diff
@@ -58,13 +59,13 @@
...
 import com.liferay.portal.kernel.search.Sort;
 import com.liferay.portal.kernel.service.ServiceContext;
+import com.liferay.portal.kernel.service.SubscriptionLocalService;
 import com.liferay.portal.kernel.social.SocialActivityManagerUtil;
 import com.liferay.portal.kernel.systemevent.SystemEvent;
 import com.liferay.portal.kernel.util.StringUtil;
 import com.liferay.portal.kernel.util.Validator;
 import com.liferay.portal.kernel.workflow.WorkflowConstants;
 import com.liferay.social.kernel.model.SocialActivityConstants;
-import com.liferay.subscription.service.SubscriptionLocalService;
```
Comment (robertoDiaz) [inline modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBThreadLocalServiceImpl.java]: Why did you change this???

Comment (hudakl) [reply modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/service/impl/MBThreadLocalServiceImpl.java]: I ran source formatter and that did it. I just committed with the fix.
